### PR TITLE
[For 2.6.3] Fix/2688 Add cypress and unit test for #2683

### PR DESCRIPTION
**Pull request by metzgegu**
#2688 

## Checkpoints

The goal of this section is to help code reviewer and merge request author to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

**For code reviewer**

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

**For developper**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [ ] Original issue have been updated with a short resume of implemented solution

**For quality team**

- [ ] Tested by quality team

### CODE REVIEW

File: functionnal_tests/cypress/integration/app_file/switch_between_file_spec.js [hunk 1]
```diff
@@ -63,4 +63,26 @@ describe('App File', () => {
...
+      cy.loginAs('administrators')
+      cy.visitPage({
+        pageName: p.CONTENT_OPEN,
+        params: { workspaceId: workspaceId, contentType: 'file', contentId: secondContentId }
+      })
+      cy.getTag({ selectorName: s.CONTENT_FRAME })
+        .get('[data-cy=popin_right_part_share]').should('be.visible')
+    })
+
+    it('should display the download share button when the user is logged as a contributor', () => {
```
Comment (Skylsmoi) [inline functionnal_tests/cypress/integration/app_file/switch_between_file_spec.js:78]: is the test name correct ?
it says it "should display" but it checks for "be.not.visible"

Comment (metzgegu) [reply functionnal_tests/cypress/integration/app_file/switch_between_file_spec.js]: No it was not correct, fixed

**Comment by PhilippeAccorsi** [comment]
@Skylsmoi @metzgegu can we test something special or not?

If not, we can merge this in 2.6.3 directly (its better to add new test quickly for all next develoment) :)

**Comment by Skylsmoi** [comment]
no, nothing special to test as long as the CI pass

**Pull request merged**